ui/components/multichain/notification-list-item-icon/notification-list-item-icon.tsx

```diff
@@ -56,6 +56,7 @@ const AvatarTokenComponent = ({ src }: { src: string }): JSX.Element => (
   <AvatarToken
     data-testid="avatar-token"
     src={src}
+    padding={1}
```

Comment:
Small padding around notification icons (gives some breathing room for large images)